### PR TITLE
fix: normalize avatar picture #1301

### LINKED ISSUE #1301: Custom avatar image is not displayed
Steps to reproduce:

1. Create 2 avatars
2. Change picture for avatar 2
3. Switch in chat to avatar 2
4. See what is displayed in avatar circles and mentions

### PR DESCRIPTION
**Pull request by frolyo**

### CODE REVIEW

File: src/dotnet/Chat.UI.Blazor/Components/Onboarding/AvatarStep.razor [hunk 1]
```diff
@@ -144,7 +146,6 @@
     }
 
     private void OnImagePicked(MediaContent mediaContent) {
-        _stepModel.Picture = mediaContent.ContentId;
```
Comment (evgeniy-ac) [inline src/dotnet/Chat.UI.Blazor/Components/Onboarding/AvatarStep.razor:147]: if OnGenerateAvatarClick was executed Picture has a link to a picture
seems like set it to "" will be useful

File: src/dotnet/Users.Service.Migration/Migrations/20230706154248_FixAvatarPictures.cs [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+                .OrderBy(c => c.Id)
+                .ToListAsync();
+            log.LogInformation("Fixing picture for {Count} avatars", dbAvatars.Count);
+            foreach (var dbAvatar in dbAvatars) {
+                var contentId = dbAvatar.Picture;
+                if (string.IsNullOrEmpty(dbAvatar.MediaId)) {
+                    var media = await mediaDbContext.Media.FirstOrDefaultAsync(x => x.ContentId == contentId);
+                    dbAvatar.MediaId = media?.Id;
+                }
+                dbAvatar.Picture = "";
```
Comment (evgeniy-ac) [inline src/dotnet/Users.Service.Migration/Migrations/20230706154248_FixAvatarPictures.cs]: Should we clear it if a media is not found?

File: src/dotnet/Chat.UI.Blazor/Components/Onboarding/AvatarStep.razor (hunk 1 above)
Comment (frolyo) [reply src/dotnet/Chat.UI.Blazor/Components/Onboarding/AvatarStep.razor:147]: done

File: src/dotnet/Users.Service.Migration/Migrations/20230706154248_FixAvatarPictures.cs (hunk 2 above)
Comment (frolyo) [reply src/dotnet/Users.Service.Migration/Migrations/20230706154248_FixAvatarPictures.cs]: done

**Pull request merged**